recipes/_configure_server.rb

```diff
@@ -193,7 +209,6 @@
     }
     allotted_pool_size += pool_max_size
   end
-  node.run_state['cerner_splunk'] ||= {}
```

Comment:
I believe having this on line 136 will guarantee it. However, based on other comments in the review, this might end up coming back here.